cmd/loadTests.go

```diff
@@ -973,22 +1018,34 @@ func userJourneyThread(frameworkMap *sync.Map, threadWaitGroup *sync.WaitGroup,
 				if err != nil {
 					logError(9, fmt.Sprintf("Unable to create the Component %s: %v", compStub.ComponentStub.ComponentName, err))
 					FailedComponentCreationsPerThread[threadIndex] += 1
+					MetricsWrapper(MetricsController, metricsConstants.CollectorComponents, metricsConstants.MetricTypeCounter, metricsConstants.MetricFailedComponentCreationCounter)
 					increaseBar(componentsBar, componentsBarMutex)
 					continue
 				}
 				if component.Name != compStub.ComponentStub.ComponentName {
 					logError(10, fmt.Sprintf("Actual component name (%s) does not match expected (%s): %v", component.Name, compStub.ComponentStub.ComponentName, err))
 					FailedComponentCreationsPerThread[threadIndex] += 1
+					MetricsWrapper(MetricsController, metricsConstants.CollectorComponents, metricsConstants.MetricTypeCounter, metricsConstants.MetricFailedComponentCreationCounter)
 					increaseBar(componentsBar, componentsBarMutex)
 					continue
 				}
 				userComponentMap.Store(username, component.Name)
 
 				ComponentCreationTimeSumPerThread[threadIndex] += componentCreationTime
+				MetricsWrapper(MetricsController, metricsConstants.CollectorComponents, metricsConstants.MetricTypeGuage, metricsConstants.MetricComponentCreationTimeGauge, float64(componentCreationTime))
```

Comment:
```suggestion
				MetricsWrapper(MetricsController, metricsConstants.CollectorComponents, metricsConstants.MetricTypeGuage, metricsConstants.MetricComponentCreationTimeGauge, componentCreationTime.Seconds())
```
Instead of nanoseconds, please push seconds.